detect/git.go

```diff
@@ -171,6 +171,8 @@ func platformFromHost(u *url.URL) scm.Platform {
 		return scm.GitHubPlatform
 	case "gitlab.com":
 		return scm.GitLabPlatform
+	case "dev.azure.com":
```

Comment:
Is there another host for ADO? I recall seeing `vs-ssh.visualstudio.com` in #1747 with a special path structure, idk if that's still relevant.

Reply:
Yups. while it should not be used anymore since depreciated. Adding visualstudio.com is easy peasy. Will do. However, there is an issue with Azure DevOps and SSH.

Remote: git@ssh.dev.azure.com:v3/foxholenl/AzureDevOps%20Projects/azure-devops-gitleaks. remote.url is empty when passed to augmentGitFinding. Will have a look but If you have any ideas @rgmz that would be great